src/state/DB.js

```diff
@@ -174,6 +171,53 @@ class DB {
     getTotalDays(sessions) {
         return sessions.filter((session, index) => index === 0 || !moment(session.start).isSame(moment(sessions[index - 1].start), 'day')).length;
     }
+
+    setUsers(users) {
+        this.users = users;
+    }
+
+    updateUsersFile() {
+        log.info('Updating users file...');
+        fs.writeFileSync(this.filename, JSON.stringify(this.users));
+    }
+
+    setConfig(config) {
+        this.config = config;
+    }
+
+    updateConfigFile() {
+        log.info('Updating config file...');
+        fs.writeFileSync(this.config_filename, JSON.stringify(this.config));
+    }
+
+    setAndUpdateConfigFile(config) {
```

Comment:
when are setConfig / setUsers every called in isolation? i think you should maybe make methods for updating specific config keys and users instead of requiring the full object to be passed because it gets really messy.